### ISSUE TITLE
Disable Opacity or change it values

### ISSUE THREAD

**Issue by 88hitman** [turlvo/flutter_animation_list#1]
This package is awesome, thanks a lot !
In your package, opacity values are hard coded like this : 
`opacity = Tween<double>(begin: 0.3, end: 1)`
Could you please give us ability ton change opacity values or just to enable/disable it ?

Thank you

**Comment by turlvo** [comment]
At first I added 'opacityRange' value for what you requested and published v2.2.1 package.
After some hours, I will also release v1 package.

If you want to disable opacity changing effect, just give a Tween value like below.

```
Tween(begin: 1.0, end: 1.0)
```


```
AnimationList(
            duration: 1500,
            reBounceDepth: 30,
            opacityRange: Tween(begin: 1.0, end: 1.0),
...
```

**Comment by turlvo** [comment]
Because I published v1, v2 package, I will close this issue.
Thank you for your interest.
